ts/src/pro/ascendex.ts

```diff
@@ -49,15 +49,23 @@ export default class ascendex extends ascendexRest {
         });
     }
 
-    async watchPublic (messageHash, params = {}) {
+    async watchPublic (messageHash, symbol: string, method = undefined, params = {}) {
```

Comment:
added arguments, similarly as it's done across other exchanges

Reply:
currently only one implementation (whitebit) accepts `method` as the argument and it is only used by `watchOrderBook` so not sure about this

Reply:
I'll clarify - this method is not going to be unified, so the similarity to other exchanges does not matter here. this specific edit was done because we needed other arguments to be passed to `watchPublic` because inside it we needed to pass those extra args to base's `.watch` method. 
hope this clarifies.